python-generation

```diff
@@ -21,13 +21,14 @@ main() {
   declare VERSION="$1"
   mkdir -p tmp
 
+  [[ ! -L Dockerfile.heroku-18 ]] && ln -s builds/heroku-18.Dockerfile Dockerfile.heroku-18
```

Comment:
Huh, how did it work without this ?

Reply:
The Python runtime generation is done manually and not very often so, I believe that the last person to do it just copied or linked the file before executing the script.